### PR TITLE
Adopt to yiisoft/html PR #52

### PR DESCRIPTION
**Pull request by vjik**
Adopt to https://github.com/yiisoft/html/pull/52

### CODE REVIEW

File: src/Menu.php [hunk 1]
```diff
@@ -356,7 +356,9 @@ private function renderItems(array $items): string
                     $lines[] = $item['label'];
                 }
             } else {
-                $lines[] = Html::tag($tag, $menu, $options);
+                $lines[] = $tag === false
```
Comment (samdark) [inline src/Menu.php:359]: Was it previously accepting `false`?

File: src/NavBar.php [hunk 2]
```diff
@@ -47,28 +47,38 @@ public function begin(): ?string
...
 
-        if (!is_string($navTag) && !is_bool($navTag) && $navTag !== null) {
-            throw new InvalidArgumentException('Tag should be either string, bool or null.');
+        if (
+            !is_string($navTag) &&
+            $navTag !== '' &&
+            !is_bool($navTag) &&
+            $navTag !== null
+        ) {
+            throw new InvalidArgumentException('Tag should be either non empty string, bool or null.');
```
Comment (samdark) [inline src/NavBar.php:56]: That's now what's in the codition.

File: src/NavBar.php [hunk 3]
```diff
@@ -47,28 +47,38 @@ public function begin(): ?string
...
         $tag = ArrayHelper::remove($this->options, 'tag', 'nav');
-        if (!is_string($tag) && !is_bool($tag) && $tag !== null) {
-            throw new InvalidArgumentException('Tag should be either string, bool or null.');
+        if (
+            !is_string($tag) &&
+            $tag !== '' &&
+            !is_bool($tag) &&
+            $tag !== null
+        ) {
+            throw new InvalidArgumentException('Tag should be either non empty string, bool or null.');
```
Comment (samdark) [inline src/NavBar.php:75]: That's not what's in the codition.

File: src/NavBar.php [hunk 4]
```diff
@@ -47,28 +47,38 @@ public function begin(): ?string
...
     }
 
     protected function run(): string
     {
         $tag = ArrayHelper::remove($this->options, 'tag', 'nav');
-        if (!is_string($tag) && !is_bool($tag) && $tag !== null) {
-            throw new InvalidArgumentException('Tag should be either string, bool or null.');
+        if (
+            !is_string($tag) &&
+            $tag !== '' &&
```
Comment (samdark) [inline src/NavBar.php:71]: Second condition doesn't make sense. If value isn't string it's not empty string by default.

**Review by samdark — CHANGES_REQUESTED** [review]

File: src/Menu.php (hunk 1 above)
Comment (vjik) [reply src/Menu.php:359]: Yes. Previous `Html::tag(false, $content)` returned pure `$content` without tag.

File: src/NavBar.php (hunk 2 above)
Comment (vjik) [reply src/NavBar.php]: Fixed and add tests.

File: src/NavBar.php (hunk 3 above)
Comment (vjik) [reply src/NavBar.php]: Fixed and add tests.

File: src/NavBar.php (hunk 4 above)
Comment (vjik) [reply src/NavBar.php]: Fixed and add tests.

**Review by samdark — APPROVED** [review]

**Pull request merged**